packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.types.ts

```diff
@@ -64,6 +64,12 @@ export interface IFocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement
    */
   firstFocusableSelector?: string | (() => string);
 
+  /**
+   * Either a full query selector for the first focusable element, or a function to select the focusable element
+   * within the area directly.
+   */
+  firstFocusableElement?: string | ((element: HTMLElement) => HTMLElement | null);
```

Comment:
now there are 2 "firstfocusable" props. And wow. That other one is terrible (called a selector (rather than firstFocusableClassName), but not actually a selector...)

If we're going to make adjustments here, let's do this properly and deprecate the old one, and support the "right" approach. I'd recommend we call this `firstFocusableTarget`, which can take an actual query selector or a React.Ref, which we could do the right thing with. (analogous to `target` in ContextualMenu I think.)